envs/env-py310.yml

```diff
@@ -75,7 +75,7 @@ dependencies:
   - nslsii >=0.8.0
   - numexpr
   - numpy >=1.20
-  - nyxtools >=0.0.10
+  - nyxtools >=0.0.11
```

Comment:
Could you please update the name `2023-1.0-py310-tiled` on top of this file too?

Reply:
done in https://github.com/nsls2-conda-envs/nsls2-collection-tiled/pull/11/commits/202e38e7b0e8034e2585d3b582776f43aca9872b